data/sql/updates/pending_db_auth/rev_1734400454914932600.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS `autobroadcast_locale`;
+CREATE TABLE `autobroadcast_locale` (
+  `realmid` INT NOT NULL,
+  `id` INT NOT NULL,
+  `locale` VARCHAR(4) NOT NULL,
+  `text` VARCHAR(45) NULL,
+  PRIMARY KEY (`realmid`, `id`));
```

Comment:
you are missing stuff.
collation, charset, db engine etc. Check your previous table implementation